### ISSUE TITLE
Replace Xerces

### ISSUE THREAD

**Issue by jberkel** [dkpro/dkpro-jwktl#28]
I'm not sure if I've run into bug #6 again, but the current wiktionary dump (20151102) does not parse correctly, failing with

```
org.xml.sax.SAXParseException; lineNumber: 99849520; columnNumber: 47; Invalid byte 2 of 4-byte UTF-8 sequence.
```

After some investigation I think I fixed the underlying issue in Xerces ([XERCES-J-1668](https://issues.apache.org/jira/browse/XERCESJ-1668)).

However the UTF-8 handling there is quite messy and has no good test coverage. I propose to replace Xerces with something else unless there are objections.

Java's default [XMLStreamReader](https://docs.oracle.com/javase/8/docs/api/index.html?javax/xml/stream/XMLStreamReader.html) could be a good option, it will probably more performant as well.

**Comment by chmeyer** [comment]
I guess removing Xerces fixed your parsing issue; which Wiktionary language version did you use for testing? It would be good if the modified JWKTL version could handle at least the latest English and German XML dump.

**Comment by jberkel** [comment]
I did fix the problem, I tested with the [English dump](http://dumps.wikimedia.org/enwiktionary/20151102/enwiktionary-20151102-pages-articles.xml.bz2).

Running 1.8.0_45 on OSX.

I'll do a test run with the German XML dump and report back.

**Comment by jberkel** [comment]
It does work with both, I added some integration tests which do a full import (not run by default, since they are obviously very slow).

**Comment by jberkel** [comment]
Closed with #29
